coverage/artifacts/profiling.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package artifacts
+
+import (
+	"os"
+	"os/exec"
+
+	"fmt"
+	"github.com/sirupsen/logrus"
+	covIo "k8s.io/test-infra/coverage/io"
+	"strings"
+)
+
+// runProfiling writes coverage profile (&its stdout) by running go test on
+// target package
+func runProfiling(cmdArgs []string, localArts *LocalArtifacts) error {
+	logrus.Info("Starts calc.runProfiling(...)")
```

Comment:
This has already drifted and become incorrect. I'd suggest not logging code paths, but their semantic meaning:


```go
logrus.Info("Starting to run profiling")
```

Reply:
Also, think about adding fields (`logrus.WithFields()`) to your logger so you add identifying metadata to your logs